map_test.go

```diff
@@ -127,6 +127,20 @@ func TestMapRedundantWithWithout(t *testing.T) {
 	}
 }
 
+func TestToFrozenMap(t *testing.T) {
+	t.Parallel()
+
+	N := 1000000
+	m := make(map[interface{}]interface{}, N)
+	for i := 0; i < N; i++ {
+		m[i] = i
```

Comment:
Make keys and values different to validate that there is no cross-contamination. `i*i` should suffice.

Reply:
changed to `i*i`